src/components/Ratings/docs/index.story.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import { Ratings, Mode, Size, Layout } from '..';
+import * as examples from './examples';
+import {
+  header,
+  api,
+  divider,
+  importExample,
+  playground,
+  tab,
+  code as baseCode,
+  tabs,
+  testkit,
+  title,
+} from 'wix-storybook-utils/Sections';
+import { allComponents } from '../../../../stories/utils/allComponents';
+import { settingsPanel } from '../../../../stories/utils/SettingsPanel';
+import * as ExtendedRawSource from '!raw-loader!./RatingsExtendedExample.tsx';
+import * as ExtendedCSSRawSource from '!raw-loader!./RatingsExtendedExample.st.css';
+import { RatingsExtendedExample } from './RatingsExtendedExample';
+
+const code = config =>
+  baseCode({ components: allComponents, compact: true, ...config });
+
+export default {
+  category: 'Components',
+  storyName: 'Ratings',
+  component: Ratings,
+  componentPath: '../Ratings.tsx',
+  componentProps: () => ({
+    'data-hook': 'storybook-Ratings',
+  }),
+  exampleProps: {
+    mode: Object.values(Mode),
+    size: Object.values(Size),
+    layout: Object.values(Layout),
```

Comment:
After fixing the `inputOptions` default value thingy, you can add the prop to the `exampleProps` like this:
```js
exampleProps: {
  ...
  inputOptions: [
    { value: [], label: 'no hovers' },
    { value: ['Very Baasa', 'Baasa', 'OK', 'Achla', 'Very Achla'], label: 'with hovers' },
  ]
  ...
}
```